CHANGELOG.md

```diff
@@ -1,4 +1,42 @@
 # Changelog
+## 1.35.1 (2025-02-24)
+### Breaking
+* Removed DS106 "is valid description name" since it was duplicated by DS104 "no description file" from the new validate. [#4774](https://github.com/demisto/demisto-sdk/pull/4774)
```

Comment:
```suggestion
* Removed *DS106* (is valid description name) since it was duplicated by DS104 (no description file) from the new validate. [#4774](https://github.com/demisto/demisto-sdk/pull/4774)
```